src/NonlinearSolve.jl

```diff
@@ -9,11 +9,12 @@ import PrecompileTools
 
 PrecompileTools.@recompile_invalidations begin
     using DiffEqBase, LinearAlgebra, LinearSolve, SparseArrays, SparseDiffTools
+    using FastBroadcast: @.., True, False
```

Comment:
`True` / `False` not used anywhere. Also they are defined by `Static.jl`, it is a bad practice to import types of a 3rd package via the 2nd